src/routes/Products/components/BasicInfoForm/index.jsx

```diff
@@ -20,7 +20,7 @@ import PT from "prop-types";
 import _ from "lodash";
 import React, { useEffect, useState } from "react";
 import DataExplorationIcon from "../../../../assets/images/data-exploration-icon.svg";
-import FindMeDataIcon from "../../../../assets/images/find-me-data-icon.svg";
+import FindMeDataIcon from "../../../../assets/images/data-advisory-icon.svg";
```

Comment:
this seems like the import is not correctly named or the file is incorrectly named.